script/vendor-mimes

```diff
@@ -2,16 +2,40 @@
 # Vendors the MIME type config from the mime-db list
 # usage: script/vendor-mimes
 
+require 'colorator'
 require 'json'
 require 'open-uri'
 
-config = File.expand_path "../lib/jekyll/mime.types", __dir__
+# ---- Helpers ----
 
-# Create an array of vendored mimetype => [extensions]
-mimes = {}
-json = URI.open('https://raw.githubusercontent.com/jshttp/mime-db/master/db.json').read
+{
+  :info    => :cyan,
+  :success => :green,
+  :error   => :red,
+}.each do |type, color|
+  define_method("log_#{type}") do |msg|
+    puts "  #{msg}".send(color)
+  end
+end
+
+# ----
+
+json = begin
+  log_info "Reading remote data.."
+  URI.open("https://raw.githubusercontent.com/jshttp/mime-db/master/db.json").read
+rescue StandardError => e
+  log_error "Error reading remote data!"
+  log_error e.message
+  log_error "Aborting."
+  exit
```

Comment:
nit: prefer `abort` when an error occurs

Reply:
I'm vetoing this because using `abort` results in Ruby 2.7 spitting out the stack trace which I don't want for a dev-dependency-script.

Reply:
The point of `abort` is to exit with the status code != 0 so that it's clear there was an error. This is a UNIX convention going back to the beginnings of UNIX. You can `echo $?` to see if the command failed and exiting with a non-zero status code will cause `&&` statements to fail.

`Kernel#exit` allows you to pass in an argument to change the exit status:

```suggestion
  exit 1
```